corehq/couchapps/last_modified/views/by_last_modified/map.js

```diff
@@ -5,9 +5,9 @@ function(doc){
         emit([doc.doc_type, doc.last_modified], null);
     } else if (doc.base_doc && (doc.base_doc === "CouchUser" || doc.base_doc === "CouchUser-Deleted")) {
         emit([doc.base_doc, doc.last_modified], null);
-    } else if (doc.doc_type === "Application" || doc.doc_type === "Application-Deleted" ||
-               doc.base_doc === "LinkedApplication" || doc.base_doc === "LinkedApplication-Deleted" ||
-               doc.base_doc === "RemoteApp" || doc.base_doc === "RemoteApp-Deleted") {
-        emit([doc.base_doc, doc.last_modified], null);
+    } else if (doc.doc_type === "Application" || doc.base_doc === "LinkedApplication" || doc.base_doc === "RemoteApp") {
+        emit(['ApplicationBase', doc.last_modified], null);
```

Comment:
i was talked out of adding this property as a `base_doc` to the `ApplicationBase` model similar to how we do with `CouchUser` (as you can see in earlier commits) because of the effort in initial migration and the risk if future subclasses will not want that property. (However, I am still open to the idea since i think it would simplify this view and many others that check for all 3 subclasses of application base)